java_console/io/src/main/java/com/rusefi/SimulatorExecHelper.java

```diff
@@ -99,10 +99,10 @@ public static void startSimulator() {
     }
 
     public static File getSimulatorBinary(String binaryPath) {
```

Comment:
@rusefillc `getSimulatorBinary` takes a base path now; the binary file name is now shared: https://github.com/FOME-Tech/fome-fw/blob/master/java_console/ui/src/main/java/com/rusefi/SimulatorHelper.java#L63